packages/manager/core/application/package.json

```diff
@@ -27,19 +27,17 @@
     "i18next": "^23.8.2",
     "i18next-http-backend": "^2.4.3",
     "react-i18next": "^14.0.5",
-    "react-router-dom": "^6.22.0",
-    "vite": "^5.2.13",
-    "vitest": "^1.2.0"
+    "react-router-dom": "^6.22.0"
   },
   "devDependencies": {
     "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "@tanstack/react-query": "5.x",
-    "@tanstack/react-query-devtools": "5.x",
-    "react": "18.x",
-    "react-dom": "18.x",
-    "react-i18next": "^14.x",
-    "react-router-dom": "^6.2x"
+    "@tanstack/react-query": "^5.37.1",
+    "@tanstack/react-query-devtools": "^5.13.3",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-i18next": "^14.0.5",
+    "react-router-dom": "^6.22.0"
```

Comment:
you have react-router-dom in both peer and deps

also i don't understand because react is in "peer dependency" in every package.json so which module is supposed to bring "react" as a dependency ? shouldn't core application have react in dependency instead of peer dependency (same for others deps declared) ?

Reply:
I will delete all the peerDependencies unecessary 👍